examples/smoke-test.sh

```diff
@@ -25,11 +25,9 @@ set -o errexit
 set -o pipefail
 
 kubectl apply -f examples/
-kubectl apply -f examples/pipelines
-kubectl apply -f examples/invocations
+kubectl apply -f examples/runs
```

Comment:
This is a typo with `run`
I think the command is this the one below. 
```
kubectl apply -f examples/run/
```

Reply:
whoops! thanks @shashwathi :sweat_smile: